src/studio/src/designer/backend/package.json

```diff
@@ -18,7 +18,7 @@
     "del": "^6.0.0",
     "gulp": "^4.0.2",
     "gulp-run-command": "^0.0.10",
-    "webpack": "^4.46.0",
+    "webpack": "^5.64.4",
```

Comment:
There might be some breaking changes between version 4 and 5 we need to handle - not sure we should update this without more investigation

Reply:
Good catch, that one slipped through. I'll revert.